job-service-container/pom.xml

```diff
@@ -397,7 +405,14 @@
                                     <Git.Commit>${git.revision}</Git.Commit>
                                 </labels>
                                 <healthCheck>
-                                    <mode>none</mode>
+                                    <!-- Check every 10 seconds -->
+                                    <interval>10s</interval>
+                                    <!-- Fail if no response after 3 seconds -->
+                                    <timeout>3s</timeout>
+                                    <!-- Fail 3 times until the container is considerd unhealthy -->
+                                    <retries>3</retries>
+                                    <!-- Command to execute in shell form -->
+                                    <cmd>curl -f http://localhost:8081/healthcheck || exit 1</cmd>
                                 </healthCheck>
```

Comment:
What needs to happen here is simply that the `healthCheck` node should be removed, so that the `HEALTHCHECK` instruction which is already in the base image is no longer overridden.

Reply:
Removed in the latest commit